common/argparse.cpp

```diff
@@ -7,11 +7,59 @@
 #include <initializer_list>
 #include <memory>
 
+#include "llvm/ADT/Sequence.h"
+#include "llvm/Support/FormatVariadic.h"
+
 namespace Carbon {
 
-Args::Arg::Arg(const ArgInfo& info) : info(info) {}
+auto CommandLine::operator<<(llvm::raw_ostream& output, ParseResult result)
+    -> llvm::raw_ostream& {
+  switch (result) {
+    case ParseResult::Error:
+      return output << "Error";
+    case ParseResult::MetaSuccess:
+      return output << "MetaSuccess";
+    case ParseResult::Success:
+      return output << "Success";
+  }
+}
+
+auto CommandLine::operator<<(llvm::raw_ostream& output, ArgKind kind)
+    -> llvm::raw_ostream& {
+  switch (kind) {
+    case ArgKind::Flag:
+      return output << "Boolean";
+    case ArgKind::Integer:
+      return output << "Integer";
+    case ArgKind::String:
+      return output << "String";
+    case ArgKind::OneOf:
+      return output << "OneOf";
+    case ArgKind::MetaActionOnly:
+      return output << "MetaActionOnly";
+    case ArgKind::Invalid:
+      return output << "Invalid";
+  }
+  CARBON_FATAL() << "Corrupt argument kind!";
```

Comment:
I'm a little confused, why these CHECKs but not for the enum on line 17-24?

Reply:
I just forgot above, added.